themes-default/slim/static/js/store.js

```diff
@@ -233,17 +233,17 @@ const store = new Puex({
     // No actions should write to the store
     // Please use store.commit to fire off a mutation that'll update the store
     actions: {
-        getConfig(store) {
+        getConfig() {
             return api.get('/config/main').then(res => {
                 store.commit(ADD_CONFIG, res.data);
```

Comment:
Did you mean to replace these stores with context as well?

Reply:
Nope, I'm using the global `store`. Just happens context also contains a small section of the store object. I think puex exports `dispatch` and `commit` only.